deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/TeletraanService.java

```diff
@@ -55,118 +49,51 @@ public void initialize(Bootstrap<TeletraanServiceConfiguration> bootstrap) {
-
-        Commits commits = new Commits(context);
-        environment.jersey().register(commits);
-
-        Deploys deploys = new Deploys(context);
-        environment.jersey().register(deploys);
-
-        Agents agents = new Agents(context);
-        environment.jersey().register(agents);
-
-        EnvAgentConfigs envAdvancedConfigs = new EnvAgentConfigs(context);
-        environment.jersey().register(envAdvancedConfigs);
-
-        EnvAgents envAgents = new EnvAgents(context);
-        environment.jersey().register(envAgents);
-
-        EnvAlarms envAlarms = new EnvAlarms(context);
-        environment.jersey().register(envAlarms);
-
-        EnvDeploys envDeploys = new EnvDeploys(context);
-        environment.jersey().register(envDeploys);
-
-        EnvCapacities envCapacitys = new EnvCapacities(context);
-        environment.jersey().register(envCapacitys);
-
-        Environs envs = new Environs(context);
-        environment.jersey().register(envs);
-
-        EnvStages envStages = new EnvStages(context);
-        environment.jersey().register(envStages);
-
-        EnvMetrics envMetrics = new EnvMetrics(context);
-        environment.jersey().register(envMetrics);
-
-        EnvHistory envHistory = new EnvHistory(context);
-        environment.jersey().register(envHistory);
-
-        EnvPromotes envPromotes = new EnvPromotes(context);
-        environment.jersey().register(envPromotes);
-
-        EnvScriptConfigs envScriptConfigs = new EnvScriptConfigs(context);
-        environment.jersey().register(envScriptConfigs);
-
-        EnvTokenRoles envTokenRoles = new EnvTokenRoles(context);
-        environment.jersey().register(envTokenRoles);
-
-        EnvUserRoles envUserRoles = new EnvUserRoles(context);
-        environment.jersey().register(envUserRoles);
-
-        EnvWebHooks envWebHooks = new EnvWebHooks(context);
-        environment.jersey().register(envWebHooks);
-
-        EnvHosts envHosts = new EnvHosts(context);
-        environment.jersey().register(envHosts);
-
-        EnvHostTags envHostTags = new EnvHostTags(context);
-        environment.jersey().register(envHostTags);
-
-        DeployConstraints deployConstraints = new DeployConstraints(context);
-        environment.jersey().register(deployConstraints);
-
-        Hotfixs hotfixes = new Hotfixs(context);
-        environment.jersey().register(hotfixes);
-
-        Ratings ratings = new Ratings(context);
-        environment.jersey().register(ratings);
-
-        SystemGroupRoles systemGroups = new SystemGroupRoles(context);
-        environment.jersey().register(systemGroups);
-
-        EnvGroupRoles envGroups = new EnvGroupRoles(context);
-        environment.jersey().register(envGroups);
-
-        Hosts hosts = new Hosts(context);
-        environment.jersey().register(hosts);
-
-        Systems systems = new Systems(context);
-        environment.jersey().register(systems);
+        environment.jersey().register(context);
```

Comment:
Is there a significant performance improvement here by not instantiating objects? At what point  is `run` executed? Wondering if it's worth measuring this somewhere